bedrock/etc/templates/etc/firefox/retention/thank-you-b.html

```diff
@@ -26,48 +28,54 @@
     <canvas id="confetti-canvas"></canvas>
     <header>
       <h1>
-        <span class="white">It's all</span></br>
-        thanks</br>
+      {% if LANG.startswith('en') %}
+        <span class="white">It’s all</span><br/>
+        thanks<br/>
         to you
+      {% else %}
+        {{ _('It’s all thanks to you') }}
+      {% endif %}
       </h1>
       <p>
-        Choosing Firefox helps Mozilla<br/>make the Internet a better place.
+        {{ _('Choosing Firefox helps Mozilla make the Internet a better place.') }}
       </p>
     </header>
   </section>
   <section class="right-panel">
     <h1>
-      Here's more super-easy things</br>
-      you can do to keep supporting</br>
-      our mission:
+      {{ _('Here’s more super-easy things you can do to keep supporting our mission:') }}
```

Comment:
Here's .... things -> Here are ... things

Reply:
Since this page is already live for en-US, I'm tempted to just ignore this and leave it as is. This would delay exposing strings, and I would get it out of the way, unless we agree quickly on fixing it.